config/settings/base.py

```
@@ -1,19 +1,23 @@
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent.parent
+load_dotenv()
 
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/6.0/howto/deployment/checklist/
+def get_env_or_raise(key: str) -> str:
+    value = os.getenv(key)
+    if not value:
+        raise RuntimeError(f"{key} 환경변수가 설정되지 않았습니다.")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)